ThunderstoreCLI/Commands/RunCommand.cs

```diff
@@ -0,0 +1,128 @@
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using ThunderstoreCLI.Configuration;
+using ThunderstoreCLI.Game;
+using ThunderstoreCLI.Utils;
+using YamlDotNet.Core.Tokens;
+
+namespace ThunderstoreCLI.Commands;
+
+public static class RunCommand
+{
+    public static int Run(Config config)
+    {
+        GameDefintionCollection collection = GameDefintionCollection.FromDirectory(config.GeneralConfig.TcliConfig);
```

Comment:
The usings were a weird idea in the first place I was testing out, it was basically used as "write the collection to a file when this goes out of scope", since Run doesn't ever edit the collection, it never needed to write, and didn't need the using. Just going to completely remove it and replace them with explicit writes though, since its a blatant misuse of the using pattern